pages/index.vue

```diff
@@ -333,6 +349,19 @@ export default {
     if (orderBy) {
       this.orderBy = orderBy
     }
+    
+    // Add `/` shortcut for search input only if not already focused
+    document.addEventListener('keypress', e => {
```

Comment:
We need to store this eventListerner to remove it on `beforeDestroy()` hook

Reply:
I added hook, a good point thanks 🙏